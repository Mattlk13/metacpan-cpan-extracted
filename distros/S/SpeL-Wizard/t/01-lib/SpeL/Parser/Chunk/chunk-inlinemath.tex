This is some inline math $y = a x^2 + bx + c$.
