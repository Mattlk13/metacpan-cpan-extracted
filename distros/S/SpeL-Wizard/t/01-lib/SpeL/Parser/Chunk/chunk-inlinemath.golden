$tex = bless( {
  'path' => 't/01-lib/SpeL/Parser/Chunk',
  'tree' => bless( {
    'ElementList' => bless( {
      'Element' => [
        bless( {
          '' => 'This is some inline math '
        }, 'SpeL::Object::TokenSequence' ),
        bless( {
          'MathUnit' => bless( {
            '' => 'y = a x^2 + bx + c',
            'Left' => bless( {
              'Component' => bless( {
                'Alphabet' => 'y',
                'trailingws' => ' '
              }, 'SpeL::Object::Variable' ),
              'Rest' => bless( {
                'ws' => ''
              }, 'SpeL::Object::Expressionrest' )
            }, 'SpeL::Object::Expression' ),
            'RelationChain' => [
              {
                'RelOperator' => bless( {
                  '' => '='
                }, 'SpeL::Object::RelOperator' ),
                'Right' => bless( {
                  'Component' => bless( {
                    'Alphabet' => 'a',
                    'trailingws' => ' '
                  }, 'SpeL::Object::Variable' ),
                  'Rest' => bless( {
                    'Op' => bless( {
                      'Op' => ''
                    }, 'SpeL::Object::Binop' ),
                    'Remainder' => bless( {
                      'Component' => bless( {
                        'Alphabet' => 'x',
                        'trailingws' => ''
                      }, 'SpeL::Object::Variable' ),
                      'Rest' => bless( {
                        'Op' => bless( {
                          'Lit' => '2',
                          'sup' => '^'
                        }, 'SpeL::Object::Power' ),
                        'Remainder' => bless( {
                          'Op' => bless( {
                            'Op' => '+'
                          }, 'SpeL::Object::Binop' ),
                          'Remainder' => bless( {
                            'Component' => bless( {
                              'Alphabet' => 'b',
                              'trailingws' => ''
                            }, 'SpeL::Object::Variable' ),
                            'Rest' => bless( {
                              'Op' => bless( {
                                'Op' => ''
                              }, 'SpeL::Object::Binop' ),
                              'Remainder' => bless( {
                                'Component' => bless( {
                                  'Alphabet' => 'x',
                                  'trailingws' => ' '
                                }, 'SpeL::Object::Variable' ),
                                'Rest' => bless( {
                                  'Op' => bless( {
                                    'Op' => '+'
                                  }, 'SpeL::Object::Binop' ),
                                  'Remainder' => bless( {
                                    'Component' => bless( {
                                      'Alphabet' => 'c',
                                      'trailingws' => ''
                                    }, 'SpeL::Object::Variable' ),
                                    'Rest' => bless( {
                                      'ws' => ''
                                    }, 'SpeL::Object::Expressionrest' )
                                  }, 'SpeL::Object::Expression' )
                                }, 'SpeL::Object::Expressionrest' )
                              }, 'SpeL::Object::Expression' )
                            }, 'SpeL::Object::Expressionrest' )
                          }, 'SpeL::Object::Expression' )
                        }, 'SpeL::Object::Expressionrest' )
                      }, 'SpeL::Object::Expressionrest' )
                    }, 'SpeL::Object::Expression' )
                  }, 'SpeL::Object::Expressionrest' )
                }, 'SpeL::Object::Expression' )
              }
            ]
          }, 'SpeL::Object::Relation' )
        }, 'SpeL::Object::MathInline' ),
        bless( {
          '' => '.

'
        }, 'SpeL::Object::TokenSequence' )
      ]
    }, 'SpeL::Object::ElementList' )
  }, 'SpeL::Object::Document' )
}, 'SpeL::Parser::Chunk' );

